proto/cosmos/sanction/v1beta1/query.proto

```diff
@@ -0,0 +1,84 @@
+syntax = "proto3";
+package cosmos.sanction.v1beta1;
+
+import "cosmos/base/query/v1beta1/pagination.proto";
+import "cosmos/sanction/v1beta1/sanction.proto";
```

Comment:
Oops. This is the old import. I'll need to update this one and a few others in these protos. Working on the fix now.

Reply:
Nope. Nevermind. These are in the `proto/cosmos/...` directory in this repo, so that's what the import needs to be too. So we're all good.